keras/layers/merge.py

```diff
@@ -187,6 +187,25 @@ class Add(_Merge):
     It takes as input a list of tensors,
     all of the same shape, and returns
     a single tensor (also of the same shape).
+
+    # Examples
+
+    ```python
+        from keras.layers.merge import Add  # a layer
+        from keras.layers.merge import add  # a functional interface
+        from keras.layers import Input, Dense
+        from keras.models import Model
+
+        input1 = Input(shape=(16,))
+        x1 = Dense(8, activation='relu')(input1)
+        input2 = Input(shape=(32,))
+        x2 = Dense(8, activation='relu')(input2)
+        added = Add()([x1, x2])
+        # added = add([x1, x2])  # which is equivalent
```

Comment:
Yeah.. but is the current one easier to understand? Well they are similar but the full line can be a bit easier for beginners I think?